<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chushi</class>
 <widget class="QMainWindow" name="chushi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTDO V1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source/pic.qrc">
    <normaloff>:/source/favicon.ico</normaloff>:/source/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_6">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(86, 128, 172);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>36</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_7">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_23">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>隶书</family>
              <pointsize>28</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 10px;
border-bottom-left-radius: 0px;
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,stop:0 rgba(255, 255, 255, 20%), stop:1 rgba(255, 255, 255, 0));</string>
            </property>
            <property name="text">
             <string>高效透平设计及优化系统</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>10</number>
            </property>
            <property name="indent">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/source/close.svg);
background-color: rgb(29, 93, 192);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8"/>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="minimumSize">
             <size>
              <width>381</width>
              <height>481</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>381</width>
              <height>481</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QFrame" name="frame_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>381</width>
               <height>481</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>381</width>
               <height>481</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>381</width>
               <height>481</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame{image: url(:/svg/未标题-1.png);

background-color: rgba(255, 255, 255, 0);}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QFrame" name="frame_3">
              <property name="geometry">
               <rect>
                <x>-10</x>
                <y>0</y>
                <width>391</width>
                <height>481</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{
	image: none;
background-color: rgba(255, 255, 255, 20%);
border-radius:6px 
}
QLabel{

	background-color: rgba(255, 255, 255, 0);
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="leftMargin">
                <number>24</number>
               </property>
               <property name="topMargin">
                <number>24</number>
               </property>
               <property name="rightMargin">
                <number>24</number>
               </property>
               <property name="bottomMargin">
                <number>8</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_10">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="spacing">
                   <number>6</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>24</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>26777</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>方正舒体</family>
                      <pointsize>36</pointsize>
                      <weight>50</weight>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>WELCOME</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_9">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <property name="spacing">
                   <number>8</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_11">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_new">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>55</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>14</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
	background-color: rgb(11, 11, 11);
	border-radius:20px ;
	color: rgb(255, 255, 255);


}</string>
                    </property>
                    <property name="text">
                     <string>分析及优化</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_new_2">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>55</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>14</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
	background-color: rgb(11, 11, 11);
	border-radius:20px ;
	color: rgb(255, 255, 255);


}</string>
                    </property>
                    <property name="text">
                     <string>设计及优化</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_read">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>55</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>14</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
	background-color: rgb(11, 11, 11);
	border-radius:20px ;
	color: rgb(255, 255, 255);


}</string>
                    </property>
                    <property name="text">
                     <string>读取现有文件</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_open">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>55</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>14</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
	background-color: rgb(11, 11, 11);
	border-radius:20px ;
	color: rgb(255, 255, 255);


}</string>
                    </property>
                    <property name="text">
                     <string>显示内置模板</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>4</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <property name="minimumSize">
             <size>
              <width>550</width>
              <height>250</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/source/leaf.png);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>24</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
            <property name="spacing">
             <number>24</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_12">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/source/xh.png);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>2023.4</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>黑体</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>西安交通大学叶轮机械研究所</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="source/pic.qrc"/>
 </resources>
 <connections/>
</ui>
